src/lib/features/traffic-data-usage/fake-traffic-data-usage-store.ts

```diff
@@ -89,4 +89,41 @@ export class FakeTrafficDataUsageStore implements ITrafficDataUsageStore {
 
         return Object.values(data);
     }
+
+    async getDailyTrafficDataForPeriod(
+        from: Date,
+        to: Date,
+    ): Promise<IStatTrafficUsage[]> {
+        return this.trafficData.filter(
+            (data) => data.day >= from && data.day <= to,
+        );
+    }
+
+    async getMonthlyTrafficDataForPeriod(
```

Comment:
"traffic data"? "traffic usage data"? got a preference?

Reply:
traffic usage * is the most accurate term, but I might have opted for keeping method names shorter some places

Reply:
Cool, I'll put the extra usage in there , then

Reply:
fixed in 9a5863b7b04015c6ebbb25083665e72f97a4c455